styles/custom.css

```diff
@@ -1,10 +1,47 @@
-/* Footer styles */
+.navbar:not(.navbar-fixed-top){
+  margin-bottom: 0;
+}
+body {
+  line-height: 1.8;
+}
+
+body .navbar *,
+body footer *
+{
+  /* Reset line-height for navbar and footer */
+  line-height: 1.42857143;
```

Comment:
That change causes that site name in navbar is slightly lower than it should be.